worlds/rabi_ribi/regions.py

```diff
@@ -0,0 +1,305 @@
+        chapter_1.place_locked_item(RabiRibiItem("Chapter 1", ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(chapter_1)
+        add_rule(chapter_1, lambda state: logic.can_reach_chapter_1(state, self.player))
+
+        chapter_2 = RabiRibiLocation(self.player, "Chapter 2", None, regions[LocationName.town_main])
+        chapter_2.place_locked_item(RabiRibiItem("Chapter 2", ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(chapter_2)
+        add_rule(chapter_2,
+                 lambda state: logic.can_reach_chapter_2(state, self.player) and
+                    state.has("Chapter 1", self.player))
+
+        chapter_3 = RabiRibiLocation(self.player, "Chapter 3", None, regions[LocationName.town_main])
+        chapter_3.place_locked_item(RabiRibiItem("Chapter 3", ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(chapter_3)
+        add_rule(chapter_3,
+                 lambda state: logic.can_reach_chapter_3(state, self.player) and
+                    state.has("Chapter 2", self.player))
+
+        chapter_4 = RabiRibiLocation(self.player, "Chapter 4", None, regions[LocationName.town_main])
+        chapter_4.place_locked_item(RabiRibiItem("Chapter 4", ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(chapter_4)
+        add_rule(chapter_4,
+                 lambda state: logic.can_reach_chapter_4(state, self.player) and
+                    state.has("Chapter 3", self.player))
+
+        chapter_5 = RabiRibiLocation(self.player, "Chapter 5", None, regions[LocationName.town_main])
+        chapter_5.place_locked_item(RabiRibiItem("Chapter 5", ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(chapter_5)
+        add_rule(chapter_5,
+                 lambda state: logic.can_reach_chapter_5(state, self.player) and
+                    state.has("Chapter 4", self.player))
+
+        bunny_strike = RabiRibiLocation(self.player, ItemName.bunny_strike, None, regions[LocationName.town_main])
+        bunny_strike.place_locked_item(RabiRibiItem(ItemName.bunny_strike, ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(bunny_strike)
+
+        speed_boost = RabiRibiLocation(self.player, ItemName.speed_boost, None, regions[LocationName.town_main])
+        speed_boost.place_locked_item(RabiRibiItem(ItemName.speed_boost, ItemClassification.progression, None, self.player))
+        regions[LocationName.town_main].locations.append(speed_boost)
+
+    def _get_region_name_list(self):
+        return [
+            convert_existing_rando_name_to_ap_name(name) for \
+            name in self.randomizer_data.graph_vertices
+        ]
```

Comment:
Didn't look at this super closely... Is there anything new here or has this just changed to a dedicated region file?